.github/workflows/release.yaml

```diff
@@ -134,6 +134,10 @@ jobs:
 
       - name: Install dependencies
         run: |
+          # We need to freeze docker.io because its update requires user input
+          sudo apt update
+          sudo apt-mark hold docker.io
+
```

Comment:
We already encountered this IIRC, there is a way to automate the user inputs also IIRC

But this works for now.

Reply:
yeah but why don't we need i in regular CIs ?

Reply:
Ho actually the thing I remember is that when this happened the quick solution that we took was to update the AMI with the newest version of `docker.io`

Reply:
interesting, but still unsure why I encounter the issue in the release CI only